pegasus/sites.v3/code.org/public/sotw.moved

```diff
@@ -1 +0,0 @@
-http://code.org/sotm
```

Comment:
Given that we are deprecating SOTM, it seems like a good time to remove the SOTW-to-SOTM redirect.